test/marketplace/Interfaces.sol

```diff
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: UNLICENSED
 
-pragma solidity 0.8.4;
+pragma solidity >= 0.8.4;
```

Comment:
Why aren't we locking in a Solidity version anymore?

Reply:
while i prefer locking to a specific version it seems that @JTraversa and everyone else in the space is fine with this. It's all over the v3 repo.